docs/getting-started/quick-start.md

```diff
@@ -6,9 +6,136 @@ sidebar_position: 2
 
 In this section we are providing a sample application that one can deploy and just use to play around.
 
-## K8S
+## Kubernetes
 
-_Here we provided a small sample on K8s_
+### Application
+
+An application is a process running within a container. Examples include a NodeJS Express application, a Ruby Rails application or a Spring Boot application.
+
+Create a NodeJS application:
+
+```yaml
+kubectl apply -f - << EOD
+---
+apiVersion: apps/v1 ############################### GROUP/VERSION
+kind: Deployment    ############################### RESOURCE
+
+metadata:
+  name: nodejs-rest-http-crud  #################### NAME
+  labels:
+    app: nodejs-rest-http-crud
+    runtime: nodejs
+    runtime-version: 14-ubi7
+
+spec:
+  selector:
+    matchLabels:
+      app: nodejs-rest-http-crud
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: nodejs-rest-http-crud
+        deploymentconfig: nodejs-rest-http-crud
+    spec:
+      containers:
+      - name: nodejs-rest-http-crud
+        image: nodejs-rest-http-crud:latest
```

Comment:
Are there any pre-requisities for this?
Are any K8S be able to pull that container? What if not?

Reply:
I have updated this section.